algorithm/eight_queens.py

```diff
@@ -8,6 +8,7 @@ def __init__(self, size: int = 8):
         self._size = size
         self._queen_positions = []
         self._board = [[{}]]
+        self.imported_boards = []
```

Comment:
pa follow nalang ng standard natin for this
```suggestion
        self._imported_boards = []
```
then gawa ka nalang ng method to extract the data
```py
@property
def imported_boards():
     return self._imported_boards
```